### ISSUE TITLE
Suggestion for good IDE setup

### ISSUE THREAD

**Issue by McFoggy** [takezoe/gitbucket#737]
Hi,

can someone give advices on how to correctly setup an IDE to work on the project?
I tried Eclipse (& scala-ide), Netbeans & IDEA without lot of success.

In all 3 I can import the project, but in none of them I have a correct setup in which I have completion & navigation between Java, Scala & templates.

Of course using sbt on command line of course I build/run correctly the project

Thanks for any help.

_PS: as you might have understand I am new to scala world_

**Comment by takezoe** [comment]
I recommend IntelliJ + Scala Plugin. You can simply open the `/gitbucket` directory as a SBT project.

**Comment by McFoggy** [comment]
Hi, thanks for the hint. 
But as I told it is exactly what I did. 
For example, opening the html templates does not lead  in having Intellij interpreting them as "scala templates". Thus completion on contollers, routes, ... does not occur. Is there anything special to configure? I think IDEA see them as standard html files. 

I come from pure java/maven world and was used to Netbeans & Eclipse. I perhaps missed something in IDEA setup but I doubt. I am using latest IDEA community edition (14.1.2) in which I activated scala support.
